framework/src/main/kotlin/br/com/zup/beagle/platform/BeaglePlatform.kt

```diff
@@ -26,7 +26,7 @@ import br.com.zup.beagle.core.ServerDrivenComponent
  */
 @Suppress("FunctionNaming")
 fun <T : ServerDrivenComponent> Platform(platform: BeaglePlatform,
-                                         self: T): BeaglePlatformWrapper<T> {
+                                                                                                                    self: T): BeaglePlatformWrapper<T> {
```

Comment:
<table data-meta="generated_by_backend">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/ZupIT/beagle-backend-kotlin/blob/852ca04da8603a3a73e6cad9cefeef6a5add212d/framework/src/main/kotlin/br/com/zup/beagle/platform/BeaglePlatform.kt#L29"></span>Line detected that is longer than the defined maximum line length in the code style.</td>
    </tr>
  </tbody>
</table>